docs/electronics/marlin-firmware.md

```diff
@@ -30,8 +30,8 @@ If you like to be more up to date with the Newest Marlin features, I have all my
 [Github pre-configured.](https://github.com/Allted/Marlin/tree/CHOOSE_VERSION)
 
 **Right now flashing Marlin 2.0 requires the [Arduino IDE 1.8.9 or
-higher](https://www.arduino.cc/en/Main/Software). Other than that follow the
-directions [below](#how-to-flash).**
+higher](https://www.arduino.cc/en/Main/Software)and follow the
+directions [below](#how-to-flash). Another option is you can now use [PlatformIO](../learn/platformio.md)**
```

Comment:
I would strike "you can now use" to it just says, "Another option is PlatformIO". The whole now/new text is going to be silly in a year.